core/src/components/select/test/basic/select.e2e.ts

```diff
@@ -150,6 +150,45 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       expect(alerts.length).toBe(1);
     });
   });
+
+  test.describe(title('select: click'), () => {
+    test('clicking a select label should only trigger onclick once', async ({ page }, testInfo) => {
```

Comment:
```suggestion
    test('should trigger onclick only once when clicking the label', async ({ page }, testInfo) => {
```

For consistency with the other tests